xblock/test/test_plugin.py

```diff
@@ -75,13 +75,13 @@ def _num_plugins_cached():
     return len(plugin.PLUGIN_CACHE)
 
 
-@XBlock.register_temp_plugin(AmbiguousBlock1, "thumbs")
```

Comment:
What was the reason for this change?

Reply:
Getting the following error otherwise
```
xblock.plugin.AmbiguousPluginError: Ambiguous entry points for thumbs: 
sample_xblocks.thumbs.thumbs.ThumbsBlock, xblock.test.test_plugin.AmbiguousBlock1
```

**Reason of occurrence:**
[Some tests](https://github.com/openedx/XBlock/blob/5de4fb9ee8215f618cbf6a551345f25583fdedcc/xblock/test/utils/test_helpers.py#L7) of the `xblock-utils` repo are using `xblock-sdk` imports due to which I have added `xblock-sdk` pip package in the requirements

After that getting conflicts with following:
[sample_xblocks.thumbs.thumbs.ThumbsBlock](https://github.com/openedx/xblock-sdk/blob/master/sample_xblocks/thumbs/thumbs.py#L97)
xblock.test.test_plugin.AmbiguousBlock1

Reply:
Gotcha, both packages are registering a plugin to the same name.  Rather than calling it `thumbs_1` I think `ambiguous_block_1` might be a better name since that's the block we're registering here.